### PR TITLE
Mixstyle refactor

### PR DESCRIPTION
**Pull request by mixmix**
this is ahdinosaurstlye-refactor with some tweaks from me .... looks like es2040 as well ..

### CODE REVIEW

File: modules_basic/thread.js [hunk 1]
```diff
@@ -121,8 +121,8 @@ exports.create = function (api) {
           meta.channel = thread[0].value.content.channel
 
           var recps = thread[0].value.content.recps
-          var private = thread[0].value.private
-          if(private) {
+          var priv = thread[0].value['private']
+          if(priv) {
```
Comment (mixmix) [inline modules_basic/thread.js]: [interest] : what does this add ?

File: modules_basic/timestamp.mcss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Timestamp {
   a {
-
+    font-size: .8rem
+    color: gray
```
Comment (mixmix) [inline modules_basic/timestamp.mcss]: hmm can we start with a simple colorscheme already. I think mcss allows mixins which would be ideal for this sort of thing right ? 
Is there a good file we can put all the colors in? that would be a themers dream!

File: modules_extra/git.js [hunk 3]
```diff
@@ -450,23 +448,53 @@ exports.create = function (api) {
...
+          c.issues ? c.issues.map(renderIssueEdit) : null)
+      }
+
+      if(c.type === 'issue') {
+        return h('p', 'opened issue on ', repoLink(c.project))
+      }
+
+      if(c.type === 'pull-request') {
+        return h('p', 'opened pull-request ',
+            'to ', repoLink(c.repo), ':', c.branch, ' ',
```
Comment (mixmix) [inline modules_extra/git.js]: indenting is a bit off

File: modules_extra/git.js [hunk 4]
```diff
@@ -450,23 +448,53 @@ exports.create = function (api) {
...
+        ])
+      }
+
+      if(c.type === 'git-update') {
+        return h('p', 'pushed to ', repoLink(c.repo))
+      }
+
+      if(c.type === 'issue-edit'
+       || (c.type === 'post' && c.text === '')) {
+        return h('div',
```
Comment (mixmix) [inline modules_extra/git.js]: lets use [ ] children everywhere for consistency and future compatability with mutant?

File: package.json [hunk 5]
```diff
@@ -68,7 +69,8 @@
   },
   "browserify": {
     "transform": [
-      "brfs"
+      "brfs",
+      "es2040"
```
Comment (mixmix) [inline package.json]: what features are we using from es2040 ? is it teh fatarrow?

File: modules_basic/message.mcss [hunk 6]
```diff
@@ -1,24 +1,59 @@
...
   border-top: solid 1px gainsboro
+  position: relative
 
   -mini {
     padding-top: 0.25rem
     padding-bottom: 0.25rem
+
+    section.content {
+      clear: none
+    }
```
Comment (ahdinosaur) [inline modules_basic/message.mcss]: it's to a mini override for `clear: both` below

File: modules_basic/message.mcss [hunk 7]
```diff
@@ -1,24 +1,59 @@
...
   header.author {
-    float: left
+    position: absolute
+    margin-left: -6rem
     display: flex
     flex-direction: column
   }
 
+  section.title {
+    float: left
```
Comment (ahdinosaur) [inline modules_basic/message.mcss]: maybe, give it a try

File: modules_basic/thread.js (hunk 1 above)
Comment (ahdinosaur) [inline modules_basic/thread.js]: es2040 (babel) errored saying that `private` is a reserved word in strict mode.

File: modules_basic/timestamp.mcss (hunk 2 above)
Comment (ahdinosaur) [inline modules_basic/timestamp.mcss]: colors module!

File: modules_extra/git.js (hunk 4 above)
Comment (ahdinosaur) [inline modules_extra/git.js]: yeah keen was mostly a copy paste

File: package.json (hunk 5 above)
Comment (ahdinosaur) [inline package.json]: fat arrow, destructuring, [const](https://github.com/ssbc/patchbay/pull/64/files#diff-669c9a9081d1b1bdfab187fda9fc386cR95), [defaults](https://github.com/ssbc/patchbay/pull/64/files#diff-d07d1f2c1a2cb988bab11232dd4d4879R24)

File: modules_basic/message.mcss (hunk 6 above)
Comment (mixmix) [inline modules_basic/message.mcss]: I'll defer to you on this, Ive actually never learnt / used clear

**Comment by mixmix** [comment]
@ahdinosaur I'm keen for either of us to execute on these changes, lets just not both make them. Hows about me?
I am keen to have a go, feels like there are a few little learning things in here. It will probably mean it will take a little longer as we're out and about with friends. If you're super keen to see it done quick, feel free to declare that and run with it. I feel like I'm learning a good amount by reviewing still

File: modules_basic/message-backlinks.js [hunk 8]
```diff
@@ -1,7 +1,19 @@
...
+exports.gives = {
+  message_backlinks: true,
+  mcss: true
+}
 
 exports.create = function (api) {
-  return function (msg) {
+  return {
+    message_backlinks,
+    mcss: () => fs.readFileSync(Path.join(__dirname, 'message-backlinks.mcss'))
```
Comment (mixmix) [inline modules_basic/message-backlinks.js]: done!

File: modules_basic/message.mcss (hunk 6 above)
Comment (mixmix) [inline modules_basic/message.mcss]: done

File: modules_basic/message.mcss (hunk 7 above)
Comment (mixmix) [inline modules_basic/message.mcss]: done

File: modules_basic/timestamp.mcss (hunk 2 above)
Comment (mixmix) [inline modules_basic/timestamp.mcss]: done - called `style-mixins`

File: modules_extra/git.js (hunk 4 above)
Comment (mixmix) [inline modules_extra/git.js]: defered

File: modules_extra/git.js (hunk 3 above)
Comment (mixmix) [inline modules_extra/git.js]: done

**Pull request merged**